frappe/integrations/doctype/google_settings/google_settings.js

```diff
@@ -3,6 +3,6 @@
 
 frappe.ui.form.on('Google Settings', {
 	refresh: function(frm) {
-		frm.dashboard.set_headline(__("For more information, {0}.", [`<a href='https://erpnext.com/docs/user/manual/en/erpnext_integration/google_settings'>${__('Click here')</a>`]));
+		frm.dashboard.set_headline(__("For more information, <a href='https://erpnext.com/docs/user/manual/en/erpnext_integration/google_settings'>Click here</a>."));
```

Comment:
Why? previous one should work... are you getting any error for that?

Reply:
Breaks in refresh hence reverting the change